p2pserver/peer/peer.go

```diff
@@ -198,86 +181,36 @@ func (this *Peer) SetHeight(height uint64) {
 	this.base.SetHeight(height)
 }
 
-//GetConsConn return consensus link
-func (this *Peer) GetConsConn() *conn.Link {
-	return this.ConsLink
-}
-
-//SetConsConn set consensue link to peer
-func (this *Peer) SetConsConn(consLink *conn.Link) {
-	this.ConsLink = consLink
-}
-
-//GetSyncState return sync state
-func (this *Peer) GetSyncState() uint32 {
-	return this.syncState
+//GetState return sync state
+func (this *Peer) GetState() uint32 {
+	return this.linkState
 }
 
 //SetSyncState set sync state to peer
 func (this *Peer) SetSyncState(state uint32) {
-	atomic.StoreUint32(&(this.syncState), state)
-}
-
-//GetConsState return peer`s consensus state
-func (this *Peer) GetConsState() uint32 {
-	return this.consState
-}
-
-//SetConsState set consensus state to peer
-func (this *Peer) SetConsState(state uint32) {
-	atomic.StoreUint32(&(this.consState), state)
+	atomic.StoreUint32(&(this.linkState), state)
 }
 
 //GetSyncPort return peer`s sync port
 func (this *Peer) GetSyncPort() uint16 {
-	return this.SyncLink.GetPort()
-}
-
-//GetConsPort return peer`s consensus port
-func (this *Peer) GetConsPort() uint16 {
-	return this.ConsLink.GetPort()
-}
-
-//SetConsPort set peer`s consensus port
-func (this *Peer) SetConsPort(port uint16) {
-	this.ConsLink.SetPort(port)
+	return this.Link.GetPort()
 }
 
 //SendToSync call sync link to send buffer
 func (this *Peer) SendToSync(msgType string, msgPayload []byte) error {
```

Comment:
remove this function.

Move all its code to Peer.SendRaw() ?

Reply:
just rename SendToSync to SendTo